src/components/Course/Course.js

```diff
@@ -43,4 +44,19 @@ const Course = ({ course }) => {
   );
 };
 
+Course.PropTypes = {
+  course: PropTypes.shape({
```

Comment:
`course` should have `isRequired` at the end of it.

Reply:
Done.